<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>1085</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 100, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topHorizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="labelsVerticalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
        <widget class="QLabel" name="balanceLabel">
         <property name="styleSheet">
          <string>font-size: 16pt; font-weight: bold; color: #d4af37;</string>
         </property>
         <property name="text">
          <string>$1000</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignBottom">
        <widget class="QLabel" name="betLabel">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string>font-size: 14pt; font-weight: bold; color: #ff4444;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>800</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>800</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="betDisplayLayout" stretch="0,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="displayChipsLeftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay1Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$1</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay5Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$5</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay10Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$10</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay25Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$25</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay50Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$50</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="betDisplay100Button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>font-size: 12pt; font-weight: bold; background-color: #2b1f0e; color: #d4af37; border: 2px solid #d4af37; border-radius: 40px;</string>
       </property>
       <property name="text">
        <string>$100</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="displayChipsRightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="betDisplayVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bottomVerticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="startButtonLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
        <widget class="QPushButton" name="startRoundButton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 16pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>Start Round</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="chipButtonsLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <spacer name="chipButtonsLeftSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="chip1Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chip5Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chip10Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chip25Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$25</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chip50Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chip100Button">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14pt; font-weight: bold;
background-color: rgb(61, 56, 70);</string>
         </property>
         <property name="text">
          <string>$100</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="chipButtonsRightSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
